test/integration/test_classification_package.py

```diff
@@ -73,4 +78,16 @@ def test_cli_3way_split(self):
         # validate label data with shapes
         self.assertEqual(data['y_train'].shape, (5, 7))
         self.assertEqual(data['y_test'].shape, (2, 7))
-        self.assertEqual(data['y_val'].shape, (1, 7))
\ No newline at end of file
+        self.assertEqual(data['y_val'].shape, (1, 7))
+
+    def test_overzoom(self):
+        """Verify data.npz produced by CLI when overzoom is used"""
+        cmd = 'label-maker package --dest integration-cl-overzoom --config test/fixtures/integration/config_overzoom.integration.json'
+        cmd = cmd.split(' ')
+        subprocess.run(cmd, universal_newlines=True)
+
+        data = np.load('integration-cl-overzoom/data.npz')
+
+        self.assertEqual(data['x_train'].shape, (6, 512, 512, 3))
+        self.assertEqual(data['x_test'].shape, (2, 512, 512, 3))
+        self.assertEqual(data['x_val'].shape, (1, 512, 512, 3))
```

Comment:
@martham93 I think this will only test that the packaging code works, rather than the new image util; we'll need to add the test for something running `label-maker images`

Reply:
is there a way to test running `label-maker images` without having a token present in a config that gets pushed up?

Reply:
actually I'll use the drone.tif!

Reply:
we can also add a token to circle and read from the environment